webpack/ForemanRhCloudHelpers.js

```diff
@@ -6,5 +6,9 @@
 export const foremanUrl = path => `${window.URL_PREFIX}${path}`;
 
 export const isNotRhelHost = ({ hostDetails }) =>
-  // eslint-disable-next-line camelcase
-  !new RegExp('red\\s?hat', 'i').test(hostDetails?.operatingsystem_name);
+  // This regex tries matches sane variations of "RedHat", "RHEL" and "RHCOS"
+  // eslint-disable-next-line spellcheck/spell-checker
```

Comment:
Does this line need to be there, it's causing issues with the test_js?

Reply:
spellcheck/spell-checker is not defined for tfm-lint, I think its better to add the word to .eslintrc.
The spelling is checked in `    "lint:spelling": "eslint ./webpack",`

Reply:
Seems like my IDE deceived me there... Adding "rhc" to the spellcheck exclusions fixed it!